.pylintrc

```diff
@@ -0,0 +1,284 @@
+# Defaults to name with leading underscore
+ignored-argument-names=_.*
+
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=120
+
+extension-pkg-allow-list=jq
+
+
+[LOGGING]
+
+# The type of string formatting that logging methods do. `old` means using %
+# formatting, `new` is for `{}` formatting.
+logging-format-style=old
+
+# Logging modules to check that the string format arguments are in logging
+# function parameter format.
+logging-modules=logging
+
+
+[MAGIC-VALUE]
+
+# List of valid magic values that `magic-value-compare` will not detect.
+# Supports integers, floats, negative numbers, for empty string enter ``''``,
+# for backslash values just use one backslash e.g \n.
+valid-magic-values=0,-1,1,,__main__
+
+
+[MESSAGES CONTROL]
+
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once). You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use "--disable=all --enable=classes
+# --disable=W".
+disable=
+    docstring-first-line-empty, # C0199
+
+# Enable the message, report, category or checker with the given id(s). You can
+# either give multiple identifier separated by comma (,) or put this option
+# multiple time (only on the command line, not in the configuration file where
+# it should appear only once). See also the "--disable" option for examples.
+enable=
+    useless-suppression
+
+
+[MISCELLANEOUS]
+
+# List of note tags to take in consideration, separated by a comma.
+notes=FIXME,XXX,TODO
+
+# Regular expression of note tags to take in consideration.
+#notes-rgx=
+
+
+[REFACTORING]
+
+# Complete name of functions that never returns. When checking for
+# inconsistent-return-statements if a never returning function is called then
+# it will be considered as an explicit return statement and no message will be
+# printed.
+never-returning-functions=sys.exit,argparse.parse_error
+
+
+[REPORTS]
+
+# Tells whether to display a full report or only the messages.
+reports=no
+
+# Activate the evaluation score.
+score=yes
+
+# Set the output format. Available formats are text, parseable, colorized, json
+# and msvs (visual studio). You can also give a reporter class, e.g.
+# mypackage.mymodule.MyReporterClass.
+output-format=colorized
+
+
+[SIMILARITIES]
+
+# Minimum lines number of a similarity.
+min-similarity-lines=4
+
+# Ignore comments when computing similarities.
+ignore-comments=no
+
+# Ignore docstrings when computing similarities.
+ignore-docstrings=no
+
+# Ignore signatures when computing similarities.
+ignore-signatures=yes
```

Comment:
Isn't most of that default option ?

Reply:
We should just keep what we want to be different than the default imo (until pylint is capable of regenerating a config file and removing the default value)

Reply:
Yes, no? 😅

idk - for pylint I always had a lot of verbose configuration.
If pylint could process my file and keep only the non-defaults ....

It seems that `pylint --generate-rcfile > .pylint-defaults` picks up _any_ available configuration on `cwd` 😕

Reply:
Yeah unfortunately we don't have the tool to make this automatically. But how about we use no pylintrc and then build from there (add only what's necessary)

Reply:
Anyway pylint is disabled on `.pre-commit`. Let's yank this too.